gammapy/makers/background/reflected.py

```diff
@@ -179,74 +159,69 @@ def _region_angular_size(pixels, center):
 
         return angular_size
 
-    def setup(self):
-        """Compute parameters for reflected regions algorithm."""
-        geom = self.reference_map.geom
-        self._pix_region = self.region.to_pixel(geom.wcs)
-        self._pix_center = PixCoord.from_sky(self.center, geom.wcs)
-
-        # Make the ON reference map
-        mask = geom.region_mask([self.region]).data
-        # on_reference_map = WcsNDMap(geom=geom, data=mask)
-
-        # Extract all pixcoords in the geom
-        X, Y = geom.get_pix()
-        ONpixels = PixCoord(X[mask], Y[mask])
+    @lazyproperty
+    def exclusion_mask_ref(self):
+        """Exclusion mask reprojected"""
+        if self.exclusion_mask:
+            mask = self.exclusion_mask.interp_to_geom(self.geom_ref, fill_value=True)
+        else:
+            mask = WcsNDMap.from_geom(geom=self.geom_ref, data=1)
```

Comment:
why not creating directly the Map as boolean?

Reply:
Good point, I tried:
```python3
m = Map.from_geom(geom=geom, data=True)
m = Map.from_geom(geom=geom, data=1, dtype=bool)
```
Neither seems to work, I'll investigate why...

Reply:
Done